### DISCUSSION TITLE
Save the style that generated an image

### DISCUSSION THREAD

**Discussion by jccluaviz** [Acly/krita-ai-diffusion#559] [Ideas]
The same way we can save the prompt that generated an image and use it to generate another one, save the style that we used to generate the image.

Thank you.

**Comment by ETCore7** [comment]
That would be nice. Fooocus saves all parameters in an HTML logfile.
![image](https://github.com/Acly/krita-ai-diffusion/assets/150020319/dcec8518-a29a-45b4-8537-05338e751527)

You probably already know this, but just in case somebody else is looking for info: As a work around you could make a new style profile when you run into something special with customized values. Via right click on a preview you can copy strength and seed. Save it with the name of the custom style in a notepad document.
![image](https://github.com/Acly/krita-ai-diffusion/assets/150020319/69312be3-ee1d-4e8d-bfdd-d412d4e9b617)

**Comment by Acly** [comment]
The difference is that styles can be changed/deleted, would you expect it to 
1. save the style name and use whatever version of the same style exists at the time?
2. save all style parameters and reuse them exactly as they were?

Anyway, number 2. is quite a bit of work, somehow need to represent temporary? zombie styles that have been changed in the meantime.

**Comment by jccluaviz** [comment]
I see. Its completely true. 

You can change the characteristics of any style at any time and thus, although the information of the style that generated the image will be stored, the result of the moment in which it was generated and the new creation with the new parameters could differ greatly. 

Although it is true that, once the characteristics of a style are defined, it will be quite rare for you to modify them in a regular way. And that makes me think that it would be equally useful to save that information. 

Anyway I understand perfectly what you are saying. 

Thank you so much.
